### ISSUE TITLE
Is it possible to train CMC by loading pretrained resnet-50 model?

### ISSUE THREAD

**Issue by AbhishekRS4** [HobbitLong/CMC#17]
I really enjoyed reading the paper. Thanks for open sourcing the code.

Please let me know how to train CMC model by loading pretrained resnet-50 trained on ImageNet.

**Comment by HobbitLong** [comment]
Hi,

If I understand correctly, are you using the pre-trained resnet-50 to do CMC on some new datasets? If so, probably you can copy the Pytorch's official ResNet and load the pre-trained weights. Then add the 128 fc layer and l2 normalization on top of it to do CMC stuff.

**Comment by AbhishekRS4** [comment]
Thanks, got it.

**Comment by AbhishekRS4** [comment]
Suppose if I want to try CMC model on custom dataset what hyperparameter change would you recommend? Thanks.

**Comment by HobbitLong** [comment]
@AbhishekRS4 , In general, I think the temperature and number of negatives are more or less stable across different datasets. Perhaps it's good to start by trying with different batch size and learning rate.
